src/data/faq_de.json

```diff
@@ -971,6 +971,16 @@
                         "textblock": [
                             "<ul><li><b>Schnelltest-Profil anlegen:</b> Um ein Schnelltest-Profil anzulegen, können Sie unter Status (vor Version 2.12 'Startseite') in Ihrer CWA auf 'Sie lassen sich testen?' tippen (bzw. auf 'Test registrieren' in App-Versionen bis 2.5 tippen). Dort finden Sie den Bereich „Schnelltest-Profil anlegen“, wo Sie Ihre persönlichen Daten, wie Name, Adresse und Telefonnummer, eintragen können. Wenn alles eingetragen ist, können Sie die Daten speichern. Die Corona-Warn-App erstellt daraufhin einen QR-Code aus den Daten, den Sie an den Teststellen vorzeigen können.</li><li><b>Schnelltest-Profil löschen (iOS):</b> Sie können ein Schnelltest-Profil löschen, indem Sie im Schnelltest-Bereich der Corona-Warn-App auf Ihr Schnelltest-Profil gehen und dort 'Bearbeiten' und dann 'Schnelltest-Profil löschen' auswählen. (In früheren Versionen erschien der Menüpunkt als 'Schnelltest-Profil entfernen'.)</li><li><b> Schnelltest-Profil löschen (Android):</b> Sie können ein Schnelltest-Profil löschen, indem Sie im Schnelltest-Bereich der Corona-Warn-App auf Ihr Schnelltest-Profil gehen und dort im 3-Punkte-Menü 'Schnelltest-Profil löschen' auswählen. (In früheren Versionen erschien dieser Menüpunkt als 'Schnelltest-Profil entfernen' bzw. als 'Entfernen'.)</li><li><b>Schnelltest-Profil bearbeiten:</b> Beabsichtigen Sie ein Schnelltest-Profil in App-Versionen bis 2.5 zu bearbeiten, können Sie das Profil zunächst entfernen (löschen) und danach ein neues Profil anlegen. Ab Version 2.6 der App kann ein Schnelltest-Profil direkt bearbeitet werden, daher ist es nicht notwendig, das Profil erst zu löschen.</li></ul>"
                         ]
+                    },
+                    {
+                        "title": "Wie effizient erkennt der von mir verwendete Test eine Infektion?",
+                        "anchor": "schnelltesttest",
+                        "active": false,
+                        "textblock": [
+                            "<a href='https://schnelltesttest.de/' target='_blank' rel='noopener noreferrer'>schnelltesttest.de</a> ist ein Open Source Projekt welches dir einen einfachen Weg bietet, Informationen über die vom Paul-Ehrlich-Institut evaluierten SARS-COV-2-Antigen-Schnelltests abzurufen.",

```

Comment:
"welches dir" => "welches Ihnen"

to be consistent with the rest of the article and the FAQs in general.

Reply:
Changed it in https://github.com/corona-warn-app/cwa-website/pull/2533/commits/3796aeb0d7637de624daf0310c8200a95d0dae96, thanks for your suggestion!